plugin/commands.go

```diff
@@ -0,0 +1,123 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
+	"gopkg.in/urfave/cli.v1"
+)
+
+// CommandPlugin is definition of mkr plugin
+var CommandPlugin = cli.Command{
+	Name:        "plugin",
+	Usage:       "Manage mackerel plugin",
+	Description: `[WIP] Manage mackerel plugin`,
+	Subcommands: []cli.Command{
+		{
+			Name:        "install",
+			Usage:       "install mackerel plugin",
+			Description: `WIP`,
+			Action:      doPluginInstall,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "prefix",
+					Usage: "plugin install location",
+				},
+			},
+		},
+	},
+	Hidden: true,
+}
+
+// main function for mkr plugin install
+func doPluginInstall(c *cli.Context) error {
+	argInstallTarget := c.Args().First()
+	if argInstallTarget == "" {
+		return fmt.Errorf("Specify install target")
+	}
+
+	err := setupPluginDir(c.String("prefix"))
+	if err != nil {
+		return errors.Wrap(err, "failed to install plugin")
+	}
+
+	_, err = parseInstallTarget(argInstallTarget)
+	if err != nil {
+		return errors.Wrap(err, "failed to install plugin")
```

Comment:
[nit]
L45 and L50 have the same error-wrapping message, imo it's better to use each wrapper should have unique message